_rules/html-page-lang-valid-bf051a.md

```diff
@@ -97,6 +97,30 @@ This `html` element has a `lang` attribute whose value is not a [valid language
 <html lang="#1"></html>
 ```
 
+#### Failed Example 3
+
+The `lang` attribute of this page is an [iso 639.2][] three letters code, hence not a [valid language tag][].
+
+```html
+<html lang="eng">
+	<body>
+		<p lang="en">I love ACT rules!</p>
```

Comment:
What's the reason for putting in the `lang` on this `p` element? I think this is clearer as an example if it isn't there. One could even make the case that in this particular example it isn't actually an error. Not that I want to have that discussion.

```suggestion
		<p>I love ACT rules!</p>
```

Reply:
Without the `lang` attribute on the paragraph, the page is also failing 3.1.2 (on top of 3.1.1 that the rule test).